web/components/navbar.module.css

```diff
@@ -17,7 +17,7 @@
   align-items: center;
   height: 50px;
   overflow: hidden;
-  background-color: #2a555e;
+  background-color: #3688b7;
```

Comment:
This makes the whole page look like https://dynabench.org/ 
I specifically did not use blue to avoid the confusion

Reply:
Reverted this!